static/blockchains/terra/projects.json

```diff
@@ -85,7 +85,7 @@
       "png": "https://assets.hermesprotocol.io/img/projects/tfm.png"
     },
     "url": "https://tfm.com/",
-    "categories": []
+    "categories": ["dex"]
```

Comment:
If we are considering TFM a DEX because people can exchange through its interface, then we should consider it an NFT marketplace as well... but not sure how truthful these two statements are, since TFM doesn't have any liquidity (either tokens or NFTs), they're just an aggregator with efficient routing mechanisms.